src/formio.form.js

```diff
@@ -92,6 +92,11 @@ const registerPlugin = (plugin) => {
       case 'evaluator':
         Formio.Evaluator.registerEvaluator(plugin.evaluator);
         break;
+      case 'default':
```

Comment:
Why is the plugin name called "default"?  This seems really strange like we should ix the place where "evaluator" is not being used.

Reply:
I understand. Going to investigate this issue some more.